src/compressed_tensors/quantization/quant_args.py

```diff
@@ -53,17 +54,30 @@ class QuantizationStrategy(str, Enum):
     TOKEN = "token"
 
 
-class ActivationOrdering(str, Enum):
+class ActivationOrdering(AliasableEnum, str, Enum):
     """
     Enum storing strategies for activation ordering
 
     Group: reorder groups and weight\n
-    Weight: only reorder weight, not groups. Slightly lower latency and
-    accuracy compared to group actorder\n
+    Weight: only reorder weight, not groups. Slightly lower accuracy but also lower
+    latency when compared to group actorder\n
+    Dynamic: alias for Group\n
+    Static: alias for Weight\n
```

Comment:
nit: remove \n

Reply:
I think this looks cleaner
<img width="667" alt="Screenshot 2024-11-26 at 13 40 27" src="https://github.com/user-attachments/assets/8d37da81-f6a6-451a-9744-68f926daac0f">

Than this
<img width="904" alt="Screenshot 2024-11-26 at 13 40 47" src="https://github.com/user-attachments/assets/33bc61da-26f2-41ef-8644-ff18f3e79120">